### PR TITLE
Intro.md

### PR DESCRIPTION
**Pull request by hthillman**

### CODE REVIEW

**Comment by hnjolles1** [comment]
That's a wrap.

File: docs/intro.md [hunk 1]
```diff
@@ -1,26 +1,67 @@
...
-new topics, add new how-to guides, and provide examples wherever you think it
-might be helpful. All documentation is editable via
-[GitHub](https://github.com/livepeer/livepeer-org/tree/master/docs).
+This site is organized by topics including concepts, tasks, and reference information designed for [Orchestrators](/livepeer-quick-start/video-mining-quick-start/choosing-a-role.md) on the Livepeer Network including:
 
-## Something missing? {#something-missing}
+- [Video Miners](/video-miners/core-concepts/roles-and-responsibilities#types-of-video-miners), including [Transcoders](/video-miners/core-concepts/roles-and-responsibilities#types-of-video-miners)
+- [Video Developers](/video-developers/overview)
+
+**Note** [Delegators](/protocol/core-concepts/ecosystem-participants#delegator) can also benefit from content on this site, with some information included for you as well.
```
Comment (hthillman) [inline docs/intro.md:16]: for some reason this sentence reads strangely for me. maybe we should omit the clause after the comma?

File: docs/intro.md [hunk 2]
```diff
@@ -1,26 +1,67 @@
...
-We are constantly improving this documentation. Here are a few ways you can make suggestions or contribute to the docs:
 
 -   Edit this article using the link at the bottom of the page. If this is your first time editing a page, please check out the [contribution guide](/contributing/overview).
 -  [File an issue](https://github.com/livepeer/docs) for us. To avoid duplicates, we ask that you do a quick search of existing issues to see if any similar issues exist.
--   Come talk to us in [Discord](https://discord.gg/uaPhtyrWsF)
\ No newline at end of file
+-   Come talk to us in [Discord](https://discord.gg/uaPhtyrWsF)
+
+
+### What's new
```
Comment (hthillman) [inline docs/intro.md:24]: love this section!

File: docs/intro.md [hunk 3]
```diff
@@ -1,26 +1,67 @@
...
+**[Latest Releases](https://github.com/livepeer/go-livepeer/releases)** We strongly encourage all node operators to upgrade to this release by February 21 so you can connect to Arbitrum Mainnet after the LIP-73 block.
+
+**Follow our roadmap** on the [Livepeer Blog](https://medium.com/livepeer-blog)
+
+**Follow us** 
+- [@LivepeerOrg](https://twitter.com/LivepeerOrg?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor) on twitter 
+- join our [community chat](https://discord.gg/RR4kFAh) on Discord.
+
+## Livepeer Quickstart
+Install and get up and running with the [Livepeer Quickstart](/installation/livepeer-quickstart) guide for Video Miners, Transcoders, and additional information for Developers.
```
[the remark this answers is not in the document]
Comment (hnjolles1) [reply docs/intro.md:41]: agreed

**Pull request merged**